clients/edgex-foundry/device_client.go

```diff
@@ -22,7 +22,11 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"net/http/cookiejar"
 	"strings"
+	"time"
+
+	"golang.org/x/net/publicsuffix"
```

Comment:
The `"golang.org/x/net/publicsuffix"` package can be put together with the third-party package below. The reference packages are divided into three parts: built-in packages, third-party packages, and packages in this repository.

Reply:
done